experimental/dds/tree2/.eslintrc.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
 			files: ["src/test/**/*"],
 			rules: {
 				"@typescript-eslint/no-unused-vars": ["off"],
+				"import/no-internal-modules": "off",
```

Comment:
should be unnecessary now

Reply:
removed `"import/no-internal-modules": "off"`